cypress/support/fragments/requests/requestDetail.js

```diff
@@ -144,6 +144,11 @@ export default {
     InteractorsTools.checkKeyValue(requesterInfoSection, 'Pickup service point', data.pickupSP);
   },
 
+  checkCreatedDate(date) {
+    cy.get('button[class^="metaHeaderButton-"]').click();
```

Comment:
Why not interactor?
For example Button(including('Record last updated')

Reply:
Done!